Cookbook/Technical-Documents/How to use Slack?.md

```diff
@@ -0,0 +1,146 @@
+Ideally follow the stackoverflow approach and state what you tried.
+
+The question should follow the format:
+`[<topic of the question>] question`
+
+Example:
+```
+[RAF] What is an example with the newest style for a page where a child triggers some events in a parent? What is best to inject, and from where?
+```
+
+### #ios-build
+
+A public channel where we usually trigger CI workflows like Testfligh/AppCenter builds or UI Automation Tests.
+The UI Automation Tests run every night and the results are published in this channel by the bot.
+To check how to trigger the workflows check [Slack CI Integration](/SlackCIIntegration.md)
+
+## How to manage notifications?
+
+Slack notifications are great to make sure we don't miss anything important. However, when they are very frequent they can disturb our work and our productivity.
+
+There are a few tips to try to reduce the number of notifications to the ones that really matter to you:
+
+### Global notification preferences
+
+To reduce the overall number of notifications we can configure the notifications on Slack doing the following:
+
+1. Go to Slack > Preferences > Notifications
+2. Configure your notifications with:
+
+<img src="Assets/onboarding/slack-notification-preferences.png">
+
+3. Go to `My keywords` and define the keywords to be used to trigger notifications. These can be your nicknames and variations of your name.
+
+4. In the `Do Not Disturb` you can set the time interval you don't want to receive any Slack notification. Ideally this should enabled during your non-working hours.
+
+On the day to day basis, if you want some quiet time, you can mute notifications temporarly by selecting the notifications icon next to your Slack organization name followed by selecting one of the options.
+
+<img src="Assets/onboarding/slack-notification-muted.png">
+
+### Leave the channel
+
+When you leave a channel you can no longer open it unless you join it again. Also, when someone shares a thread from that channel with you, you will have to join the channel to be able to read it.
+When the channel is private, if you leave it you will have to be invited by one of its members to join it again.
+
+You should leave a channel when:
+
+ - You haven't open it in the last month.
+ - The subject is no longer relevant or obsolete.
+
+### Mute the channel
+
+Muting a channel means you will have access to it but you won't be notified if you are mentioned individually or with `@channel` or `@here`.
+
+You should mute a channel when:
+
+ - The subject is not your main focus but you like to keep yourself updated on what is happening.
+ - It is a private channel and you don't want to lose access to it, but you don't check it on a day to day basis.
+
+
+## How to communicate my schedule?
+
+Your status on Slack should be a way to communicate what is your current status. This can be used to indicate you are working remotely, you are in a meeting or any other status.
```

Comment:
```suggestion
Your status on Slack should be a way to communicate what is your current status. This can be used to indicate you are working remotely, in a meeting, or any other statuses.
```